jenkins/nginx.sls

```diff
@@ -23,3 +25,18 @@ extend:
         - file: /etc/nginx/sites-available/jenkins.conf
       - require:
         - file: /etc/nginx/sites-enabled/jenkins.conf
+
+{% else %}
+
+Add nginx config for jenkins:
+  file.managed:
+    - template: jinja
+    - name: {{ jenkins.nginx_vhost_path }}/jenkins.conf
```

Comment:
I didn't know how to handle that, it is a required value, I can set to default such as /etc/nginx/conf.d for example, but it is only going to run when symlink_vhost is False, not sure if it should be documented or assume a default